app/src/main/kotlin/uk/govuk/app/ui/GovUkApp.kt

```diff
@@ -183,7 +188,9 @@ private fun BottomNavScaffold(
             color = GovUkTheme.colourScheme.surfaces.background
         ) {
             val startDestination =
-                if (shouldDisplayAnalyticsConsent) {
+                if (shouldDisplayUnavailable) {
```

Comment:
I've introduced an app launch navigation class to tidy this up. Might be worth waiting until my changes are merged and then integrating this with it.

Reply:
Done